src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -15,16 +15,18 @@ void EnCow_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnCow_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnCow_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_8099CAA8(EnCow* this, GlobalContext* globalCtx);
-void func_8099CB20(EnCow* this, GlobalContext* globalCtx);
-void func_8099CB68(EnCow* this, GlobalContext* globalCtx);
-void func_8099CBCC(EnCow* this, GlobalContext* globalCtx);
-void func_8099CC68(EnCow* this, GlobalContext* globalCtx);
-void func_8099CCF8(EnCow* this, GlobalContext* globalCtx);
-void func_8099CDA0(EnCow* this, GlobalContext* globalCtx);
-void func_8099CFAC(EnCow* this, GlobalContext* globalCtx);
-
-#if 0
+void EnCow_TalkEnd(EnCow* this, GlobalContext* globalCtx);
+void EnCow_GiveMilkEnd(EnCow* this, GlobalContext* globalCtx);
+void EnCow_GiveMilkWait(EnCow* this, GlobalContext* globalCtx);
+void EnCow_GiveMilk(EnCow* this, GlobalContext* globalCtx);
+void EnCow_CheckForEmptyBottle(EnCow* this, GlobalContext* globalCtx);
+void EnCow_Talk(EnCow* this, GlobalContext* globalCtx);
+void EnCow_Idle(EnCow* this, GlobalContext* globalCtx);
+void EnCow_DoTail(EnCow* this, GlobalContext* globalCtx);
+
+void EnCow_UpdateTail(Actor* this, GlobalContext* globalCtx);
```

Comment:
Any particular reason for this gap? If you're going to put a gap, I think it would be better to put the action function with the rest of the tail stuff.

Reply:
The gap was for keeping all the action functions together, but I don't mind either way. Both make sense to me. I'll change it.